### ISSUE TITLE
Cloud SQL Proxy returns 404

### ISSUE THREAD

**Issue by kothariaditya** [GoogleCloudPlatform/cloudsql-proxy#346]
When I try to run `cloud_sql_proxy -instances XXXX:XX-XXXX:XXXX-postgres=tcp:XXX` I get a 404:
```
googleapi: got HTTP response code 404 with body: <!DOCTYPE html>
  <p>The requested URL <code>/sql/v1beta4/sql/v1beta4/projects/XXXXX</code> was not found on this server.  <ins>That’s all we know.</ins>
```

This error does not occur when I use v1.16. Only occurs for the latest master.

**Comment by kurtisvg** [comment]
A few questions for trying to replicate:
1. How are you trying to compile? What commands and version of go are you using?
2. When you say "v1.16" - are you using the compiled version (downloaded from the releases page), or the commit at that tag?
3. Are you using the `go.mod` file in the root of the proxy? Can you verify which version of `google.golang.org/api` you are using?

**Comment by kothariaditya** [comment]
Sorry for the late response!
1. The go version is go1.13.5. I'm just running a script which calls `cloud_sql_proxy -instances XXXX:XX-XXXX:XXXX-postgres=tcp:XXX`
2. I believe I'm using the commit at that tag. Basically I'm doing `gco 1.16` and then ` go get github.com/GoogleCloudPlatform/cloudsql-proxy/cmd/cloud_sql_proxy`. 
3. Yup the `go.mod` file is in the root of the proxy. For `google.golang.org/api`,  a simple `git show` tells me this is the latest commit `b6b5836d7694afe29d5b84d9a4c15e3d7c2677f4`. Let me know if I can dig in further on this.

Not sure if this is related but I thought this PR may have something to do with the bug: https://github.com/GoogleCloudPlatform/cloudsql-proxy/pull/338/files#diff-2e191ab6bb0c70e18327a004be427a10L75

I see the same `v1beta4` being mentioned.

Let me know how I can help debug this further. Thanks!

**Comment by kurtisvg** [comment]
Yes, the problem is that tag`1.16` and the version of `google.golang.org/api` is `b6b5836d7694afe29d5b84d9a4c15e3d7c2677f4` are imcompatible. Unfortunately `google.golang.org.api` had non-backwards compatible change, which we fixed in PR #338. 

You'll either have to roll back to an earlier version of `google.golang.org/api` or use a commit off the master branch past eb5e09ef70a515db75cc59caaee50d29638ad5db

**Comment by kothariaditya** [comment]
Got it, thanks @kurtisvg!
